clients/cloud/python/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip curl vim
+RUN pip3 install -U confluent-kafka
+RUN pip3 install -U confluent-kafka[avro]
```

Comment:
Minor: is it more "pythonic" to include a `requirements.txt` or [pipfile](https://github.com/pypa/pipfile).

Reply:
Thanks @russau .   @javabrett was making a similar point in Slack.  Please see latest commit that moves to` requirements.txt`